its/ruling/src/test/resources/expected/python-S5655.json

```diff
@@ -1,7 +1,4 @@
 {
-'project:buildbot-0.8.6p1/contrib/svn_watcher.py':[
-119,
```

Comment:
This one was a TP for an old version of python but a FP for a most recent version. We ignore now the win32